tests/test_evented_model.py

```diff
@@ -845,3 +848,50 @@ class Model(EventedModel):
         m.a = 3
     check_mock.assert_has_calls([call(Model, "a", 3, 1)])
     mock1.assert_called_once()
+
+
+def test_single_emit():
```

Comment:
This test name needs a more descriptive name and/or a docstring to explain what it's testing. Maybe considering moving it under `test_derived_events` as well, since it is related.

Reply:
I think that edge case should have own test function